internal/impl/mongodb/cdc/input.go

```diff
@@ -276,6 +298,14 @@ type mongoBatch struct {
 	ackFn     service.AckFunc
 }
 
+type documentMode int
+
+const (
+	documentModePreAndPostImage = iota
```

Comment:
Shouldn't this be `documentModePreAndPostImage documentMode = iota`?

Reply:
Ah yes thanks. Good catch

Reply:
Really wish Go didn't have these implicit conversions...